src/Microsoft.AspNetCore.SignalR.Common/Protocol/IHubProtocol.cs

```diff
@@ -7,20 +7,54 @@
 
 namespace Microsoft.AspNetCore.SignalR.Protocol
 {
+    /// <summary>
+    /// A protocol abstraction for communicating with SignalR hubs.
+    /// </summary>
     public interface IHubProtocol
     {
+        /// <summary>
+        /// Gets the name of the protocol. The name is used by SignalR to resolve the protocol between the client and server.
+        /// </summary>
         string Name { get; }
 
+        /// <summary>
+        /// Gets the version of the protocol.
+        /// </summary>
         int Version { get; }
 
+        /// <summary>
+        /// Gets the transfer format of the protocol.
+        /// </summary>
         TransferFormat TransferFormat { get; }
 
+        /// <summary>
+        /// Creates a new <see cref="HubMessage"/> using the specified bytes and binder.
```

Comment:
Creates a new [HubMessage} ~~using the specified bytes and~~ **from the specified serialized representation, and using the specified** binder